src/components/testCaseList/TestCaseList.tsx

```diff
@@ -89,14 +87,8 @@ const TestCaseList = () => {
   }, [measure]);
 
   useEffect(() => {
-    setCanEdit(
-      measure?.createdBy === userName ||
-        measure?.acls?.some(
-          (acl) =>
-            acl.userId === userName && acl.roles.indexOf("SHARED_WITH") >= 0
-        )
-    );
-  }, [measure, userName]);
+    setCanEdit(checkUserCanEdit(measure?.createdBy, measure?.acls));
```

Comment:
same as on EditTestCase - this seems like we're checking if the measure creator has edit permissions. Shouldn't we check against the currently logged in user?

Reply:
@nmorasb Same as above, thanks so much Nate.

Reply:
@nmorasb the logic is here for your reference: https://github.com/MeasureAuthoringTool/madie-util/pull/25/files

Reply:
gotcha